scipy/signal/tests/test_filter_design.py

```diff
@@ -3611,6 +3611,83 @@ def test_fs_param(self):
         assert_allclose(abs(hp[2]), 1, rtol=1e-10)
 
 
+class TestIIRComb(object):
+    # Test erroneus input cases
+    def test_invalid_input(self):
+        # w0 is <= 0 or >= fs / 2
+        fs = 1000
+        with pytest.raises(ValueError, match='w0 must be between '):
+            iircomb(-fs, 30, 4, fs=fs)
+            iircomb(0, 35, 5, fs=fs)
+            iircomb(fs / 2, 40, 6, fs=fs)
+            iircomb(fs, 35, 7, fs=fs)
+
+        # Filter type is not notch or peak
+        with pytest.raises(ValueError, match='ftype must be '):
+            iircomb(0.2, 30, 4, ftype='natch')
+            iircomb(0.5, 35, 5, ftype='comb')
+
+        # Order is <= 0 or > 24
+        with pytest.raises(ValueError, match='N must be '):
+            iircomb(440, 30, -50, ftype='notch', fs=16000)
+            iircomb(220, 35, 0, ftype='peak', fs=32000)
+            iircomb(110, 40, 25, ftype='peak', fs=44100)
+            iircomb(55, 35, 50, ftype='notch', fs=48000)
+
+    # Verify that the filter's frequency response contains a
+    # notch or peak at the cutoff frequency
+    def test_frequency_response(self):
+        ftypes = ['notch', 'peak']
```

Comment:
nowadays we can `@pytest.parametrize('ftypes', ('notch', 'peak')` the `test_frequency_response(self, ftypes)` function

Reply:
Fixed.